utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -152,9 +152,9 @@
 //!
 //! Similarly, each pallet can expose a function in `#[pallet::hooks]` section as follows:
 //!
-//! ```
-//! #[cfg(feature = try-runtime)]
-//! fn try_state(_) -> Result<(), &'static str> {}
+//! ```ignore
+//! #[cfg(feature = "try-runtime")]
```

Comment:
The CI did not catch this, but locally it failed when doing `cargo test -p try-runtime-cli --features try-runtime`.